windows/wrapper/impl_webrtc_VideoCapturer.cpp

```diff
@@ -757,12 +759,13 @@ namespace webrtc
   bool CaptureDevice::FilterMediaType(int width,
                                       int height,
                                       double framerate) const {
-    RTC_CHECK(media_capture_);
     if ((width_ > 0) && (width != width_))
       return true;
     if ((height_ > 0) && (height != height_))
       return true;
-    if ((framerate_ > 0) && (framerate != framerate_))
+    // Compare framerate with 2 decimals, to distinguish between 29.97 FPS
+    // and 30.0 FPS.
+    if ((framerate_ > 0) && (std::fabs(framerate - framerate_) >= 0.01))
```

Comment:
>framerate_ > 0 [](start = 9, length = 14)

How about checking against negative "framerate" passed as parameter?

Reply:
`framerate` is coming from Media Foundation, I do hope no video capture device out there is ever advertising itself as supporting any negative framerate! (yes I can fix just in case).